spyderlib/widgets/arraybuilder.py

```diff
@@ -202,11 +206,11 @@ def __init__(self, parent, inline=True, offset=0):
         if inline:
             self._button_help.setToolTip(self._help_inline)
             self._text = NumpyArrayInline(self)
-            self._widget = self._text
+            self.widget = self._text

```

Comment:
I agree with you for runtime coed, but I tend to see tests a bit differently. Sometimes you have to do weird things in tests (monkeypatching, mocks, and so on) so accessing a private variable isn't a big deal.